_includes/logo.md

```diff
@@ -0,0 +1,22 @@
+{%- assign platform = include.platform | downcase -%}
+{%- case platform -%}
+    {%- when "c#" -%}
+        {%- assign platform = "dotnet" -%}
+    {%- when "kotlin" -%}
+        {%- assign platform = "android" -%}
+{%- endcase -%}
+{%- assign file_name = platform | prepend: '/' | append: '.svg' -%}
+{%- for file in site.static_files -%}
+    {%- if file.path contains '/assets/img/logos/' and file.path contains file_name -%}
+![{{ include.platform }}](/assets/img/logos/{{ platform }}.svg)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-image] Use the trailing [] on reference images](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5659803)